src/Tribe/Plugins_API.php

```diff
@@ -282,6 +282,21 @@ public function get_products() {
 				'free'            => false,
 				'active_installs' => 2500,
 			],
+			'event-tickets-wallet-plus' => [
+				'title'            => __( 'Wallet Plus', 'tribe-common' ),
+				'slug'             => 'event-tickets-wallet-plus',
+				'link'             => 'https://evnt.is/1bd7',
+				'plugin-dir'       => 'event-tickets-wallet-plus',
+				'main-file'        => 'event-tickets-wallet-plus.php',
+				'description'      => __( 'Allow event attendees to save their tickets to digital wallets, like Apple Wallet or PassWallet, and download PDF tickets, to have them with them at any time.', 'tribe-common' ),
+				'description-help' => __( 'Allow event attendees to save their tickets to digital wallets, like Apple Wallet or PassWallet, and download PDF tickets, to have them with them at any time.', 'tribe-common' ),
```

Comment:
This is my bad. Now that I see it in place (thanks for the artifacts🌮) it looks a bit out of balance with the others, and taking too much space. I'll adjust the scope of the ticket to reflect this.

```suggestion
				'description-help' => __( 'Allow event attendees to save their tickets to digital wallets.', 'tribe-common' ),
```